crmsh/ui_cluster.py

```diff
@@ -29,13 +29,60 @@ def parse_options(parser, args):
         options, args = parser.parse_known_args(list(args))
     except:
         return None, None
-    if options.help:
+    if hasattr(options, 'help') and options.help:
         parser.print_help()
         return None, None
     utils.check_space_option_value(options)
     return options, args
 
 
+def parse_option_for_nodes(context, *args):
+    """
+    Parse option for nodes
+    Return a node list
+    """
+    action_type = context.get_command_name()
+    action_target = "node" if action_type in ["standby", "online"] else "cluster service"
+    action = "{} {}".format(action_type, action_target)
+    usage_template = """
+Specify node(s) on which to {action}.
+If no nodes are specified, {action} on the local node.
+If --all is specified, {action} on all nodes."""
+    addtion_usage = ""
+    if action_type == "standby":
+        usage_template += """
+\n\nAdditionally, you may specify a lifetime for the standby---if set to
+"reboot", the node will be back online once it reboots. "forever" will
+keep the node in standby after reboot. The life time defaults to
+"forever"."""
+        addtion_usage = " [lifetime]"
+
+    parser = ArgParser(description=usage_template.format(action=action),
+            usage="{} [--all | <node>... ]{}".format(action_type, addtion_usage),
+            add_help=False,
+            formatter_class=RawDescriptionHelpFormatter)
+    parser.add_argument("-h", "--help", action="store_true", dest="help", help="Show this help message")
+    parser.add_argument("--all", help="To {} on all nodes".format(action), action="store_true", dest="all")
+
+    options, args = parse_options(parser, args)
+    if options is None or args is None:
+        raise utils.TerminateSubCommand
+    if options.all and args:
+        context.fatal_error("Should either use --all or specific node")
```

Comment:
s/node/node(s)/

Ideally, the same for several other places.

Reply:
Done